src/streaming/utils/RequestModifier.js

```diff
@@ -31,20 +31,39 @@
 
 import FactoryMaker from '../../core/FactoryMaker';
 
+export function modifyRequest(httpRequest, requestModifier) {
+    const request = {
+        url: httpRequest.url,
+        method: httpRequest.method,
+        headers: Object.assign({}, httpRequest.headers),
+        credentials: httpRequest.withCredentials ? 'include' : undefined,
```

Comment:
What is the reason to not use `credentials` here and `withCredentials` with a boolean?

Reply:
The idea is that object passed to the `modifyRequest` function is based on the browser's [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request) object.